### PR TITLE
pr2-interface.l: add :go-pos-unsafe, :go-pos-unsafe-no-wait, :go-pos-unsafe-wait

### PR DESCRIPTION
**Pull request by k-okada**
please check this on real robot @h-kamada

### CODE REVIEW

**Comment by furushchev** [comment]
Did you check on real robot?
I suspect pr2 always returns nil from `:go-pos-unsafe`.

**Comment by k-okada** [comment]
Oh, I didn't know specification/convention of return value.
please update document section of https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/163/files, then we'll fix the pr2-interface.l program

File: pr2eus/pr2-interface.l [hunk 1]
```diff
@@ -595,11 +600,28 @@
...
+         (setq step (/ 1000.0 msec))
          (ros::ros-info "                diff-pos ~A ~A, diff-angle ~A" x y d)
-         )
+
+         (if (and (<= (sqrt (+ (* x x) (* y y)))  0.025)
+                  (<= (abs d) 2.5) ;; 3 cm and 3deg
+                  (/= counter 0))  ;; try at least 1 time
+             (progn
+               (setq go-pos-unsafe-goal-msg nil) ;; go-pos-unsafe-wait has been called
+               (return-from :go-pos-unsafe-wait nil))
```
Comment (furushchev) [inline pr2eus/pr2-interface.l]: why returns `nil`? I assume `t` here

Comment (h-kamada) [inline pr2eus/pr2-interface.l]: Sorry, I think so too.

Comment (furushchev) [inline pr2eus/pr2-interface.l]: @h-kamada Thanks for review. Created another PR.

**Pull request merged**